### DISCUSSION TITLE
Зубы

### DISCUSSION THREAD

**Discussion by dosinabox** [dosinabox/g2nr_unofficial_update#361] [Ideas]
В Г1 пять текстур зубов:
![HUM_TEETH_V0-C](https://user-images.githubusercontent.com/39790055/77853505-1616c400-71ed-11ea-898c-d6dc60634892.png)
![HUM_TEETH_V1-C](https://user-images.githubusercontent.com/39790055/77853506-16af5a80-71ed-11ea-9dd0-a2ecd528f668.png)
![HUM_TEETH_V2-C](https://user-images.githubusercontent.com/39790055/77853507-16af5a80-71ed-11ea-8eb6-e3d26f0b8c7b.png)
![HUM_TEETH_V3-C](https://user-images.githubusercontent.com/39790055/77853508-1747f100-71ed-11ea-83e4-14f6a65968e9.png)
![HUM_TEETH_V4-C](https://user-images.githubusercontent.com/39790055/77853510-1747f100-71ed-11ea-85ff-4d217dfead51.png)

В Г2 осталась только первая текстура. При этом в Г2 есть персонажи из Г1, где у них были другие зубы:

текстура 2:
Бладвин, Нефариус, Кронос, Риордиан, Фортуно, Вольф, Милтен, Корд, Джарвис, Торлоф, Декстер, Ксардас

текстура 3:
Ворон, Скатти, Сифер, Лестер, Фиск, Кавалорн

текстура 5:
Ларес, Миксир, Диего

Текстуру 4 использовали зомби, а в Г2 у них чистые белые зубы)

**Comment by Kor-Angar** [comment]
> текстура 5:
> Ларес, Миксир, Диего

вот сколько раз играл, удалось один раз заметить у Ларса во время диалога.

а у зомби разве зубы будут видны? на сколько я понял зубы видны только во время разговора.. или нет?

**Comment by dosinabox** [comment]
> вот сколько раз играл, удалось один раз заметить у Ларса во время диалога.

Даже на 4К мониторе зубы видны с трудом и очень редко:
<img width="186" alt="Screenshot_2" src="https://user-images.githubusercontent.com/39790055/77929313-2f357880-72b2-11ea-836d-b7caec1ea352.png">
<img width="203" alt="Screenshot_1" src="https://user-images.githubusercontent.com/39790055/77929322-3066a580-72b2-11ea-967f-deb7b375bd7f.png">

> а у зомби разве зубы будут видны?

Нет, их рты никогда не открываются.

**Comment by Kor-Angar** [comment]
Тогда на Зомби можно забить.Заметнее всего должно быть у Ларса и Диего 🙃

**Comment by Kor-Angar** [comment]
Табличка :-)

Текстура | NPC | Картинка
-- | -- | --
2 | Бладвин, Нефариус, Кронос,   Риордиан, Фортуно, Вольф, Милтен, Корд, Джарвис, Торлоф, Декстер, Ксардас | ![HUM_TEETH_V1-C](https://user-images.githubusercontent.com/39790055/77853506-16af5a80-71ed-11ea-9dd0-a2ecd528f668.png)
3 | Ворон, Скатти, Сифер, Лестер,   Фиск, Кавалорн | ![HUM_TEETH_V2-C](https://user-images.githubusercontent.com/39790055/77853507-16af5a80-71ed-11ea-8eb6-e3d26f0b8c7b.png)
4 |зомби | ![HUM_TEETH_V3-C](https://user-images.githubusercontent.com/39790055/77853508-1747f100-71ed-11ea-83e4-14f6a65968e9.png)
5  | Ларес , Миксир, Диего | ![HUM_TEETH_V4-C](https://user-images.githubusercontent.com/39790055/77853510-1747f100-71ed-11ea-85ff-4d217dfead51.png)
1 | остальные | ![HUM_TEETH_V0-C](https://user-images.githubusercontent.com/39790055/77853505-1616c400-71ed-11ea-898c-d6dc60634892.png)

**Comment by dosinabox** [comment]
> очень редко

![Screenshot_5](https://user-images.githubusercontent.com/39790055/78499362-ecafe800-7758-11ea-9dd4-31d6623a8192.jpg)

:)

**Comment by Kor-Angar** [comment]
чем история закончилась?

**Comment by Kor-Angar** [comment]
у зомби Равена зубы "DEFAULT"  стоят

Mdl_SetVisualBody(self,"Zom_Body",0,0,"Zom_Head",0,DEFAULT,ITAR_GRD_H);

**Comment by dosinabox** [comment]
Пока без новостей.
И нужно будет править модель головы Hum_Head_Pony, потому что у нее перепутаны материалы:

![20201001222832_1](https://user-images.githubusercontent.com/39790055/97225746-1a877780-17e4-11eb-85a1-e91c341919e6.jpg)
![20201001222644_1](https://user-images.githubusercontent.com/39790055/97225749-1b200e00-17e4-11eb-8fed-73aa20e2017b.jpg)

**Reply by dosinabox** [reply]
исправлено

**Reply by dosinabox** [reply]
то же самое у Hum_Head_Thief

**Comment by Kor-Angar** [comment]
можно добавить зубы просто паре НПС прописав в инстанции явно Mdl_SetVisualBody

то есть лучше добавить зубы трем грубо говоря трём НПС и добавить это в релиз, получив обратную реакцию. чем это тема бедет болтаться годами.

**Reply by dosinabox** [reply]
Добавлены всем в соответствии с табличкой вверху. Работает.

**Reply by Kor-Angar** [reply]
Круто. И зомбякам? 😀

**Reply by dosinabox** [reply]
И зомбякам.
Но никому из персонажей Г2. Потом можно будет.

**Comment by dosinabox** [comment]
https://github.com/dosinabox/g2nr_unofficial_update/commit/ec73938fc75190025f9b786e2c52dc0446688347
